modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -880,17 +893,25 @@ Resources:
               mode: '000400'
               owner: root
             /etc/cfn/scripts/watchmaker-install.sh:
-              content: !Sub |
-                  #!/bin/bash
-                  PYPI_URL=${PypiIndexUrl}
+              content:
+                !If
+                  - UseWamStandalone
+                  - !Sub |
+                      #!/bin/bash
+                      curl --fail -sSL -o /etc/cfn/scripts/watchmaker ${WatchmakerStandaloneUrl}
+                      chown root:root /etc/cfn/scripts/watchmaker
```

Comment:
Stupid question but if root's the one downloading the file doesn't root own it by default? Or is there some other weirdness going on here that I'm just not seeing?

Reply:
It _shouldn't_ be necessary, but was just covering the bases to ensure we have permission to the file. Plus, I was copying/pasting from other similar approaches in the template. :D

Reply:
Ah gotcha, figured as much just wanted to make sure.